scripts/variantstore/wdl/extract/scale_xy_bed_values.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+import sys
+
+
+def scale_xy_bed_values(input_file, output_file, x_scale_factor, y_scale_factor):
+    input_bed = open(input_file, 'r')
+    output_bed = open(output_file, 'w')
+
+    fail = False
+
+    if x_scale_factor < 1.0:
+        print(f"Error: illegal X chromosome weight scale factor {x_scale_factor}; scale factor value must be >= 1.0")
+        fail = True
```

Comment:
Why not just sys.exit(1) here (and one line 17).

Reply:
to be able to report both errors if both are wrong

Reply:
Ah, gotcha